CDC-Data-Reconciliation-Backend/server.py

```diff
@@ -191,6 +211,12 @@ async def automatic_report(year: int, cdc_file:  UploadFile = File(None)):
     process = await asyncio.create_subprocess_exec(sys.executable, os.path.join(app.dir, "compare.py"), '-c', cdc_save_to, '-s', state_save_to, '-o', res_file)
     await process.wait()
 
+    if not os.path.exists(archive_path):
+        os.makedirs(archive_path)
+
+    # writing the newly created results file to the archive folder too
+    shutil.copy2(res_file, archive_path)
```

Comment:
Again make sure we are creating another folder within the archive_path with a unique ID or renaming each results file with a unique ID so that the files are not overwritten every time a report is ran.